spec/requests/landing_page_spec.rb

```diff
@@ -20,26 +34,5 @@
         expect(response).to render_template(:show)
       end
     end
-
-    context "when a content item does not exist" do
```

Comment:
I think this should be left in for now, until the scaffold is completely removed.

Reply:
As above.